site/index.html

```diff
@@ -82,7 +82,7 @@ <h2>Cloudproof Encryption</h2>
     <div class="menu">
         <button onclick="location.href='findex.html';">FINDEX</button>
         <button onclick="location.href='cloudproof.html';">CLOUDPROOF ENCRYPTION</button>
-        <!-- <button onclick="location.href='abe.html';">ABE DEMO</button> -->
+        <button onclick="location.href='abe.html';">ATTRIBUTE BASED ENCRYPTION</button>
```

Comment:
Why not using link here ?

Reply:
there is a CSS over `button`, that is the main reason imho